spec/features/view_ip_address_readiness_spec.rb

```diff
@@ -0,0 +1,52 @@
+# TODO: don't require eveerything ever to fill in a form
+require 'features/support/not_signed_in'
+require 'features/support/sign_up_helpers'
+require 'features/support/errors_in_form'
+require 'features/support/activation_notice'
+require 'support/notifications_service'
+require 'support/confirmation_use_case'
```

Comment:
Of the above 7 lines of `require`s, I think you only need `require 'features/support/sign_up_helpers'`. You're not using `it_behaves_like` on any of the others.